.travis.yml

```diff
@@ -11,6 +11,7 @@ matrix:
     - sudo: 9000
       dist: trusty
       script:
+         - ./mach clean-nightlies --keep 2 --force
```

Comment:
Let's put this into `before_cache` instead of `script`. Docs: https://docs.travis-ci.com/user/caching/#before_cache-phase